lib/assert.js

```diff
@@ -966,21 +966,23 @@ assert.ifError = function ifError(err) {
       // This will remove any duplicated frames from the error frames taken
       // from within `ifError` and add the original error frames to the newly
       // created ones.
-      const tmp2 = StringPrototypeSplit(origStack, '\n');
-      ArrayPrototypeShift(tmp2);
-      // Filter all frames existing in err.stack.
-      let tmp1 = StringPrototypeSplit(newErr.stack, '\n');
-      for (const errFrame of tmp2) {
-        // Find the first occurrence of the frame.
-        const pos = ArrayPrototypeIndexOf(tmp1, errFrame);
-        if (pos !== -1) {
-          // Only keep new frames.
-          tmp1 = ArrayPrototypeSlice(tmp1, 0, pos);
-          break;
+      const origStackStart = origStack.indexOf('\n    at');
```

Comment:
```suggestion
      const origStackStart = StringPrototypeIndexOf(origStack, '\n    at');
```

Reply:
Thanks for the suggestion but this is not mandated by our linter. Since I fail to see how this will benefit users, I would rather not include that change.